etc/openqa/openqa.ini

```diff
@@ -128,3 +128,5 @@ blacklist = job_grab job_done
 
 [misc_limits]
 #untracked_assets_storage_duration = 14
+#screenshot_cleanup_batch_size = 200000
+#screenshot_cleanup_batches_per_minion_job = 450
```

Comment:
You might want to add some hints to the instance admin how to select the right value

Reply:
I can give a generic hint *what* these parameters configure. However, I will not add any estimations like I did in the previous code comments (removed in this PR) because they will likely just not be true in practice anyways.

Reply:
I've added a comment stating what these parameters change and the effect of increasing/decreasing them.